### DISCUSSION TITLE
终端显示 CJK 字符乱码

### DISCUSSION THREAD

**Discussion by FrzMtrsprt** [1715173329/UnblockNeteaseMusic#570] [Q&A] [answered]
如图，歌名中含有 CJK 字符时，会显示乱码，命令提示符和 Windows Terminal 中均会出现此问题。

![WindowsTerminal_isthmd7p6O](https://user-images.githubusercontent.com/15852717/157228011-aad4f873-801c-4216-aa6d-54ec6e06146e.png)

（请忽略 yt-dlp 的报错

**Comment by pan93412** [comment, marked as answer]
這是 Windows 終端預設不支援 Unicode 編碼所致。應不屬於 UnblockNeteaseMusic 的問題。

**Reply by FrzMtrsprt** [reply]
原版 UnblockNeteaseMusic 没有此问题，是不是可以考虑将编码从 GBK 换回 UTF-8，毕竟 Powershell 和 cmd 的默认编码都是 UTF-8

**Reply by pan93412** [reply]
> 原版 UnblockNeteaseMusic 没有此问题，是不是可以考虑将编码从 GBK 换回 UTF-8，毕竟 Powershell 和 cmd 的默认编码都是 UTF-8

預設 PowerShell 和 cmd 的編碼 **不是** UTF-8 吧。[這應該是 pino 的已知問題，暫時無解。](https://github.com/pinojs/pino/blob/master/docs/help.md#windows)

**Reply by FrzMtrsprt** [reply]
原来如此，了解。

**Reply by byludy** [reply]
运行代码的前面加上 “chcp 65001”  CMD就会改成UTF-8编码 回显就可以正常显示
